tracer/test/Datadog.Trace.DuckTyping.Tests/GetAssemblyTests.cs

```diff
@@ -47,13 +49,26 @@ public void GetAssemblyTest()
              * WARNING: This number is expected to change if you add
              * a another test to the ducktype assembly.
              */
+            if (!CIVisibility.IsRunning)
+            {
+#if NETFRAMEWORK
+                asmDuckTypes.Should().Be(1131);
+#elif NETCOREAPP2_1
+                asmDuckTypes.Should().Be(1134);
+#else
+                asmDuckTypes.Should().Be(1135);
+#endif
+            }
+            else
+            {
```

Comment:
I assume this is because of CI visibility running in-process in this case, and generating its own duck types?

```suggestion
            // When running inside CI Visibility, we will generate additional duck types
            {
```

Reply:
done in [3844b97](https://github.com/DataDog/dd-trace-dotnet/pull/5325/commits/3844b97f9598b9c1de1a08caa8f2c1ba13a15979)